homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -0,0 +1,98 @@
+"""Support for Aurora ABB PowerOne Solar Photvoltaic (PV) inverter."""
+
+import logging
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party import groups.

Reply:
Updated